### ISSUE TITLE
xView 2018 Detection Challenge 20GB

### ISSUE THREAD

**Issue by andreaschandra** [jakartaresearch/earth-vision#21]
Add object detection dataset
https://github.com/chrieke/awesome-satellite-imagery-datasets#2-object-detection

- [xView 2018 Detection Challenge](http://xviewdataset.org/) (DIUx, Jul 2018)
60 categories from helicopter to stadium, 1 million instances, Worldview-3 imagery (0.3m res.), COCO data format, pre-trained Tensorflow and Pytorch baseline models, Paper: Lam et al. 2018

The training dataset contains 846 high-resolution images, which occupy approximately 14 GB disk space as a zip archive and 23 GB uncompressed. There are 281 validation images, which occupy approximately 5 GB compressed and 8 GB uncompressed.

https://challenge.xviewdataset.org/data-download must login

**Comment by andreaschandra** [comment]
sample at gs://ossjr/xview

**Comment by dwiuzila** [comment]
image sizes are different, any ideas on how to treat them and the bounding boxes?

**Comment by andreaschandra** [comment]
@dwiuzila can be done using padding and adjust bbox coordinate

**Comment by andreaschandra** [comment]
@dwiuzila this is rare to have images with multiple size. there may something to do in handling the size on data loader. So we don't have to put large pad on small images to fit with the largest size on dataset. the pad is given only to fit the size on the current batch

**Comment by juannat7** [comment]
this should be handled by users, can improve on this in the future if the need is there.
